ngraph/frontend/onnx/frontend/src/input_model.cpp

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <frontend_manager/frontend_exceptions.hpp>
+#include <onnx_frontend/input_model.hpp>
+#include <onnx_frontend/place.hpp>
+
+using namespace ngraph;
+using namespace ngraph::frontend;
+
+InputModelONNX::InputModelONNX(const std::string& path)
+    : m_editor(path)
+{
+}
+
+std::vector<Place::Ptr> InputModelONNX::get_inputs() const
+{
+    auto inputs = m_editor.model_inputs();
+    std::vector<Place::Ptr> ret;
+    ret.reserve(inputs.size());
+    for (const auto& input : inputs)
+    {
+        ret.push_back(std::make_shared<PlaceTensorONNX>(input, m_editor));
+    }
+    return ret;
+}
+
+Place::Ptr InputModelONNX::get_place_by_tensor_name(const std::string& tensor_name) const
```

Comment:
Will these methods implement some use case for Model Optimizer conversion? I can't find `Place::is_equal` implementation which is mandatory for options like '--input'

Reply:
It will be done in separate PR